### ISSUE TITLE
Authentication Error with every non-root user

### ISSUE THREAD

**Issue by MarcusCaepio** [Napsty/check_esxi_hardware#29]
Hi Claudio,
using ESXi 6.5 I am currently fighting with Authgentication Errors.
I want to use a monitoring user and what I did so far was:
Create a user in vSphere Web Client with admin rights and also create a user local on the esxi, also with admin rights. Both gave me an authentication error and only the root user works.
Can you give me a hint how to fix this? I want to use a read-only user.
Anyway thanks again for the great plugin.

**Comment by Napsty** [comment]
Hello @MarcusCaepio 
Did you already try https://www.claudiokuenzler.com/blog/114/check-esxi-wbem-esxi-4.1-user-authorization ? Running a non-root user in the root group is currently your best chance. Not sure though, if this possibility was removed in 6.5 (don't have 6.5 servers to test).

But yes, I would prefer a non-root user or a user which is not in the root group, too. I've already contacted VMware at least twice in the past year about this. Nobody seems to care over there.
